### PR TITLE
Adding Button

### PR DESCRIPTION
**Pull request by AdrianBor**

### CODE REVIEW

**Comment by lukaswelte** [comment]
You have a problem with the styling on mobile 
img_1599

File: src/pages/FilterMenu.css [hunk 1]
```diff
@@ -117,18 +117,23 @@ ul.FilterMenu-trendy li.FilterMenu-trendy-title {
...
-  font-size: 3em;
-  line-height: 1.5em;
-  text-align: center;
-  color: #FFFFFF;
-  border-radius: 35px;
-  background: url(../img/btn-check.png) no-repeat center center;
-  order: 0;
-  flex: 0 1 auto;
-  align-self: center;
+.FilterMenu-valid {
```
Comment (lukaswelte) [inline src/pages/FilterMenu.css]: that is a button so maybe just call it checkButton or so but maybe not valid. I would assume valid to be a class that handles the style of a input field once it is valid or something.

**Comment by AdrianBor** [comment]
@lukaswelte I saw that already on the first version, do you have an idea of where the problem is? Because size is defined already...

**Comment by AdrianBor** [comment]
@lukaswelte Ok I changed some options. Could you tell me ? (I've no access to the pre visualisation for that one)

**Comment by lukaswelte** [comment]
Don't know. It could be that some css attribute is not supported by mobile Safari. You can check that in caniuse.com

But for debugging you can remote debug the safari on your mobile [there are many tutorials how you enable it](https://blog.idrsolutions.com/2015/02/remote-debugging-ios-safari-on-os-x-windows-and-linux/).

File: src/pages/FilterMenu.css [hunk 2]
```diff
@@ -117,18 +117,23 @@ ul.FilterMenu-trendy li.FilterMenu-trendy-title {
...
-  flex: 0 1 auto;
-  align-self: center;
+.FilterMenu-checkButton {
+width: 70px;
+height: 70px;
+border-radius: 40px;
+border:0;
+cursor: pointer;
+text-align: center;
+font-size: 4em;
```
Comment (lukaswelte) [inline src/pages/FilterMenu.css]: why do I need font-sizes and line heights when the content is just a background image?

File: src/pages/FilterMenu.css [hunk 3]
```diff
@@ -117,18 +117,23 @@ ul.FilterMenu-trendy li.FilterMenu-trendy-title {
...
-  color: #FFFFFF;
-  border-radius: 35px;
-  background: url(../img/btn-check.png) no-repeat center center;
-  order: 0;
-  flex: 0 1 auto;
-  align-self: center;
+.FilterMenu-checkButton {
+width: 70px;
+height: 70px;
+border-radius: 40px;
```
Comment (lukaswelte) [inline src/pages/FilterMenu.css]: If you want to have it be 100% round the border-radius needs to be half the width and height. So 35px in this case

**Review by lukaswelte — CHANGES_REQUESTED** [review]

File: src/pages/FilterMenu.css (hunk 2 above)
Comment (AdrianBor) [reply src/pages/FilterMenu.css]: ok @lukaswelte , changements adjusted, tell me if it Changed it or not on mobile version.

**Comment by lukaswelte** [comment]
And with your last commit the button is perfect round on iOS as well, that was the thing ;)

**Review by lukaswelte — APPROVED** [review]

**Comment by AdrianBor** [comment]
Great :)

**Pull request merged**